integration/ct_integration_test.go

```diff
@@ -43,34 +46,52 @@ func TestCTIntegration(t *testing.T) {
 	fmt.Printf("Today's test has been brought to you by the letters C and T and the number %#x\n", *seed)
 	rand.Seed(*seed)
 
-	cfg, err := ctfe.LogConfigFromFile(*logConfigFlag)
+	cfgs, err := ct.LogConfigFromFile(*logConfigFlag)
 	if err != nil {
 		t.Fatalf("Failed to read log config: %v", err)
 	}
 
-	type result struct {
-		prefix string
-		err    error
-	}
-	results := make(chan result, len(cfg))
-	var wg sync.WaitGroup
-	for _, c := range cfg {
-		wg.Add(1)
-		go func(c ctfe.LogConfig) {
-			defer wg.Done()
+	for _, cfg := range cfgs {
```

Comment:
if cfgs weren't loaded from a file, this could be a proper table run test. 

```
 for _, test := range struct { 
// test vars
} {
{ //test case },
} {
// test code
}